### PR TITLE
clear redundant code

### PR DESCRIPTION
**Pull request by linlinnn**
### Modifications
clear redundant code

### CODE REVIEW

**Review by freeznet — APPROVED** [review]

**Review by aahmed-se — APPROVED** [review]

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java [hunk 1]
```diff
@@ -1048,12 +1048,12 @@ public PulsarClient getReplicationClient(String cluster) {
                     clientBuilder.serviceUrl(
                             isNotBlank(data.getBrokerServiceUrl()) ? data.getBrokerServiceUrl() : data.getServiceUrl());
                 }
-                if (data.getProxyProtocol() != null && StringUtils.isNotBlank(data.getProxyServiceUrl())) {
```
Comment (sijie) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:1051]: I think we need this `data.getProxyProtocol() != null` check, no?

Comment (linlinnn) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:1051]: StringUtils.isNotBlank contains this check, if `data.getProxyProtocol()` is null, then return false

Comment (sijie) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java:1051]: they are two different settings. One is `getProxyProtocol` and the other one is `getProxyServiceUrl`

Comment (linlinnn) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java]: oh, sorry, I will fix it.

**Review by hangc0276 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**